README.md

```diff
@@ -1,3 +1,74 @@
 # Conan Visual Studio Extension
 
-Visual Studio Extension compatible with Conan>=2.0
+![Build](https://github.com/conan-io/conan-vs-extension/actions/workflows/main.yml/badge.svg?branch=develop2)
+
+Conan, the C and C++ package manager, is now directly integrated into your IDE. List
+available libraries, install, and use them without leaving Visual Studio.
+
+## Using the Extension
+
+> **Note**
+> 
+> For a detailed explanation of how to use the extension, please visit [our documentation
```

Comment:
The text says extension, but this just links to the general docs. Maybe a wording change?

Reply:
Maybe we can use this link: https://docs.conan.io/2/integrations/visual_studio.html as we have to update these docs to also documento the use of the extension?